pom.xml

```diff
@@ -4,53 +4,151 @@
 		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 
-	<groupId>org.snomed.otf</groupId>
-	<artifactId>classification-service</artifactId>
-	<version>7.1.0-SNAPSHOT</version>
-
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
 		<version>2.1.3.RELEASE</version>
 	</parent>
 
+	<groupId>org.snomed.otf</groupId>
+	<artifactId>classification-service</artifactId>
+	<version>7.0.0</version>
+
 	<properties>
 		<java.version>11</java.version>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<maven.compiler.source>${java.version}</maven.compiler.source>
 		<maven.compiler.target>${java.version}</maven.compiler.target>
 		<packageName>${project.artifactId}</packageName>
 		<execFinalName>${project.build.finalName}.jar</execFinalName>
+		<spring-boot.version>${project.parent.version}</spring-boot.version>
+
 		<spring-cloud-aws-version>1.2.4.RELEASE</spring-cloud-aws-version>
 		<slf4j.version>1.7.35</slf4j.version>
 		<log4j.version>2.17.2</log4j.version>
+		<snomed-owl-toolkit.version>3.0.9</snomed-owl-toolkit.version>
+		<otf-common.version>2.2.0</otf-common.version>
+		<resource-manager.version>2.2.0</resource-manager.version>
+		<jaxb-api.artifactId>jaxb-api</jaxb-api.artifactId>
+		<springfox-swagger2.version>2.5.0</springfox-swagger2.version>
+		<springfox-swagger-ui.version>2.5.0</springfox-swagger-ui.version>
 	</properties>
 
+	<dependencyManagement>
+		<dependencies>
+			<!-- SNOMED OWL Toolkit (does all the heavy lifting) -->
+			<dependency>
+				<groupId>org.snomed.otf</groupId>
+				<artifactId>snomed-owl-toolkit</artifactId>
+				<version>${snomed-owl-toolkit.version}</version>
+			</dependency>
+			<!-- Snomed International common -->
+			<dependency>
+				<groupId>org.ihtsdo.otf.common</groupId>
+				<artifactId>otf-common</artifactId>
+				<version>${otf-common.version}</version>
+			</dependency>
```

Comment:
It is a nice idea but I don't think using <dependencyManagement> tag in this service offers the same benefit as in PIP-206.

Reply:
I agree, but this is with the idea of creating a BOM in the future, so I've made the pom's consistent.